trulens_eval/trulens_eval/feedback_prompts.py

```diff
@@ -182,3 +182,14 @@
 
 Answer only with an integer from 1 to 10 based on how close the responses are to the right answer.
 """
+
+LANGCHAIN_CONCISENESS_PROMPT =  "Is the submission concise and to the point? Respond only as a number from 1 to 10 where 1 is the least concise and 10 is the most concise."
```

Comment:
Is there a way for us to import these from langchain as opposed to copying these in?